UPGRADE-1.9.md

```diff
@@ -1,5 +1,52 @@
 # UPGRADE FROM `v1.8.X` TO `v1.9.0`
 
+### Package upgrades
+
+1. We've removed the support for Symfony's Templating component (which is removed in Symfony 5). 
+
+    * Remove `templating` from framework's configuration:
+        
+        ```diff
+        # config/packages/framework.yaml
+       
+        framework:
+            # ...
+        -    templating: { engines: ["twig"] }
+        ```
+      
+    * Replace any usages of `Symfony\Bundle\FrameworkBundle\Templating\EngineInterface` with `Twig\Environment`.
+   
+        Inject `twig` service into your controllers instead of `templating`.
+      
+        `$templating->renderResponse(...)` might be replaced with `new Response($twig->render(...))`.
+    
+1. We've upgraded Sylius' ResourceBundle and GridBundle packages which forced us to upgrade major versions of our dependencies.
```

Comment:
```suggestion
1. We've upgraded Sylius ResourceBundle and GridBundle packages which forced us to upgrade major versions of our dependencies.
```

Reply:
I'll fix that in other PR as in this one I wanted only to change the sections order